contracts/ScheduledPaymentModule.sol

```diff
@@ -136,7 +134,7 @@ contract ScheduledPaymentModule is Module {
         uint256 executionGas,
         uint256 maxGasPrice,
         address gasToken,
-        uint256 _nonce,
+        string memory nonce,
```

Comment:
i'm confused why nonce is a string and not a number (uint256)

Reply:
I believe adding the integer nonce is a remnant from times where we thought we care about the order of scheduled payments, also it might just be copied from the zodiac modules we were looking at

But later we discussed the question if we even need the nonce and we said that it's still good to have it so that it can act as salt for the payment hash. The safe owner can then just use a random string for this (or a UUID). That's why it's a string now. Maybe it would be better to rename to `salt` to remove any confusion?

Reply:
Agreed. I think `salt` better describes the purpose.

Reply:
Renamed nonce to salt